_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_ddi.cpp

```diff
@@ -441,14 +441,15 @@ namespace MfxHwVP9Encode
         {
             if (task.m_insertIVFSeqHeader)
             {
-                AddSeqHeader(task.m_frameParam.width,
-                    task.m_frameParam.height,
-                    par.mfx.FrameInfo.FrameRateExtN,
-                    par.mfx.FrameInfo.FrameRateExtD,
-                    0,
-                    localBuf.pBuffer,
-                    bufferSizeBytes);
-                ivfHeaderSize += IVF_SEQ_HEADER_SIZE_BYTES;
+            	mfxStatus sts = AddSeqHeader(task.m_frameParam.width,
+                                             task.m_frameParam.height,
+                                             par.mfx.FrameInfo.FrameRateExtN,
+                                             par.mfx.FrameInfo.FrameRateExtD,
+                                             0,
+                                             localBuf.pBuffer,
+                                             bufferSizeBytes);
+            	if (sts == MFX_ERR_NONE)
```

Comment:
no, if it returns error, I think we should stop the execution. i.e. use macro MFX_CHECK_STS (or just write manual code if don't have this macro in scope)